Jenkinsfile

```diff
@@ -124,8 +124,8 @@ node {
         stage("clang-debug:addr-ub-sanitizers") {
           if (env.BRANCH_NAME == 'master' || full_ci) {
             sh "export CCACHE_BASEDIR=`pwd`; cd clang-debug-addr-ub-sanitizers && make hyriseTest hyriseSystemTest -j \$(( \$(cat /proc/cpuinfo | grep processor | wc -l) / 3))"
-            sh "LSAN_OPTIONS=suppressions=.lsan-ignore.txt ASAN_OPTIONS=suppressions=.asan-ignore.txt ./clang-debug-addr-ub-sanitizers/hyriseTest clang-debug-addr-ub-sanitizers"
-            sh "LSAN_OPTIONS=suppressions=.lsan-ignore.txt ASAN_OPTIONS=suppressions=.asan-ignore.txt ./clang-debug-addr-ub-sanitizers/hyriseSystemTest clang-debug-addr-ub-sanitizers"
+            sh "LSAN_OPTIONS=suppressions=artifacts/ci/.lsan-ignore.txt ASAN_OPTIONS=suppressions=artifacts/ci/.asan-ignore.txt ./clang-debug-addr-ub-sanitizers/hyriseTest clang-debug-addr-ub-sanitizers"
```

Comment:
From a quick look, the `/ci` folder was the only one where I had no idea what it would do. For me, sanitization and CI have little in common. I think those three files don't necessarily need their own subsubfolder.

Reply:
All of these files are somehow related to sanitization or checks. How about a subfolder `checkers_and_sanitizers`?

Reply:
moved the ignore-files to `resources/`, because, yeah, why have a folder for 4 files in the first place. The important part is that they're out of view from the root directory.